x-pack/filebeat/module/salesforce/login/_meta/fields.yml

```diff
@@ -0,0 +1,119 @@
+- name: salesforce.login
+  type: group
+  # TODO: What should be the appropriate release value?
+  release: ga

```

Comment:
Definitely not a GA, let's start with "experimental" or "beta"

Reply:
Got it, thanks!